### PR TITLE
Errata: Fix typos & `\s+$`

### PR DESCRIPTION
**Pull request by tolgacangoz**
Fixes typos and trims trailing whitespaces.

@stevhliu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Please refer to the [How to use Stable Diffusion in Apple Silicon](https://huggi
 
 ## Quickstart
 
-Generating outputs is super easy with 🤗 Diffusers. To generate an image from text, use the `from_pretrained` method to load any pretrained diffusion model (browse the [Hub](https://huggingface.co/models?library=diffusers&sort=downloads) for 27.000+ checkpoints):
+Generating outputs is super easy with 🤗 Diffusers. To generate an image from text, use the `from_pretrained` method to load any pretrained diffusion model (browse the [Hub](https://huggingface.co/models?library=diffusers&sort=downloads) for 30.000+ checkpoints):
```
Comment (stevhliu) [inline README.md:70]:
```suggestion
Generating outputs is super easy with 🤗 Diffusers. To generate an image from text, use the `from_pretrained` method to load any pretrained diffusion model (browse the [Hub](https://huggingface.co/models?library=diffusers&sort=downloads) for 30,000+ checkpoints):
```

File: README.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ Also, say 👋 in our public Discord channel <a href="https://discord.gg/G7tWnz9
 - https://github.com/deep-floyd/IF
 - https://github.com/bentoml/BentoML
 - https://github.com/bmaltais/kohya_ss
-- +12.000 other amazing GitHub repositories 💪
+- +14.000 other amazing GitHub repositories 💪
```
Comment (stevhliu) [inline README.md:212]:
```suggestion
- +14,000 other amazing GitHub repositories 💪
```

**Review by stevhliu — APPROVED** [review]
Cool, thanks for the fixes!

**Review by stevhliu — APPROVED** [review]

**Comment by tolgacangoz** [comment]
Thanks for merging!

**Pull request merged**